evmbin/src/main.rs

```diff
@@ -78,39 +78,42 @@ EVM implementation for Parity.
   Copyright 2015-2019 Parity Technologies (UK) Ltd.
 
 Usage:
-    parity-evm state-test <file> [--json --std-json --std-dump-json --only NAME --chain CHAIN --std-out-only --std-err-only]
+    parity-evm state-test <file> [--chain CHAIN --only NAME --json --std-json --std-dump-json --std-out-only --std-err-only]
     parity-evm stats [options]
     parity-evm stats-jsontests-vm <file>
     parity-evm [options]
     parity-evm [-h | --help]
 
 Commands:
-    state-test         Run a state test from a json file.
+    state-test         Run a state test on a provided state test JSON file.
     stats              Execute EVM runtime code and return the statistics.
-    stats-jsontests-vm Execute standard json-tests format VMTests and return
-                       timing statistics in tsv format.
+    stats-jsontests-vm Execute standard json-tests on a provided state test JSON
+                       file path, format VMTests, and return timing statistics
+                       in tsv format.
 
 Transaction options:
     --code CODE        Contract code as hex (without 0x).
     --to ADDRESS       Recipient address (without 0x).
     --from ADDRESS     Sender address (without 0x).
     --input DATA       Input data as hex (without 0x).
     --gas GAS          Supplied gas as hex (without 0x).
-    --gas-price WEI    Supplied gas price as hex (without 0x).
+    --gas-price GWEI   Supplied gas price as hex (without 0x).
```

Comment:
This may be a backward incompatible change. I have a feeling maybe keeping `WEI` is better because this is supposed to be really raw interface (how do you specify a test requiring 2.1 GWEI gas?).

Reply:
thanks i've pushed update in commit 5575db7140ce2101ef4a15197c000a7f20f4fe80